src/main/resources/application.properties

```diff
@@ -70,6 +70,9 @@ notification.sender.address=
 spring.mail.host=
 spring.mail.port=
 
+# Kafka service properties.
+kafka.consumers.cache.size=50
```

Comment:
Could you please add this into PR description, easier to scrap it during release